js/client.js

```diff
@@ -1471,7 +1471,7 @@
 			if (leftWidth < leftMinMain) {
 				leftWidth = leftMinMain;
 			}
-			if (this.curRoom.type === 'battle' && this.sideRoom.type === 'chat') {
+			if (this.curRoom.type === 'battle' && this.sideRoom.type !== 'battle') {
```

Comment:
They do, but they also can't be side rooms, as far as I can tell, which is why I figured this should be fine to avoid a long and unwieldy condition there. To my knowledge, only chats, battles, the rooms tab and the battles tab can be on the right side. I tried to test all possible combinations, and nothing seemed to get broken.